packages/commonwealth/server/routes/writeUserSetting.ts

```diff
@@ -2,7 +2,13 @@ import { AppError } from 'common-common/src/errors';
 import type { NextFunction, Request, Response } from 'express';
 import type { DB } from '../models';
 
-const VALID_DIGEST_INTERVALS = ['daily', 'never'];
+const VALID_DIGEST_INTERVALS = [
+  'never',
+  'weekly',
+  'daily',
+  'biweekly',
```

Comment:
not sold on biweekly for every 2 weeks. @jnaviask for the best solution here

Reply:
2weeks?

Reply:
We need to be clear on what biweekly means here. 

TBC: biweekly can mean either "twice a week" or "every 2 weeks" (don't blame me I didn't do that) but is more commonly used for latter. 

However I kind of have a niggle here in that the option ordering makes no sense: either biweekly or daily should go between never/weekly, but the above order here doesn't reflect a sequence that makes sense for either definition.